<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>461</width>
     <height>381</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 18px</string>
   </property>
  </widget>
  <widget class="QListWidget" name="taskListWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>230</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 27px
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>1031</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font-size: 60px;
background-color: #7958C0;
border-radius:15px;</string>
   </property>
   <property name="text">
    <string>Daily Tasks</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="savebutton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>580</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
color: white;
background-color: #CE9D31;
border-radius:10px;
font-weight: bold;

</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
color: white;
background-color: #CE9D31;
border-radius:10px;
font-weight: bold;

</string>
   </property>
   <property name="text">
    <string>Add new</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>195</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15px;
color: white;
background-color: #CE9D31;
border-radius:10px;
font-weight: bold;

</string>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
